### ISSUE TITLE
v2.3.3 WatchCardHeroVideo Parser error: Cannot read properties of undefined (reading 'accessibilityData')

### ISSUE THREAD

**Issue by burhan-syed** [LuanRT/YouTube.js#230]
### Steps to reproduce

1. Call .search() with query that will include a watch_card in the results (for example 'mkbhd')
2. Function will crash

To reproduce in tests replace first video in test/constants.ts with: 
```
 {
    ID: 'I1qsF0WQy8c',
    QUERY: 'mkbhd',
  }  
```

### Failure Logs

```shell
InnertubeError: Something went wrong at WatchCardHeroVideo!
    This is a bug, please report it at https://github.com/LuanRT/YouTube.js/issues
        at Function._Parser_printError (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:482:7)
        at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:382:27)
        at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:430:39)
        at new UniversalWatchCard (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/UniversalWatchCard.ts:15:34)
        at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:377:24)
        at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:417:29)
        at new SecondarySearchContainer (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/SecondarySearchContainer.ts:11:28)
        at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:377:24)
        at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:430:39)
        at new TwoColumnSearchResults (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/TwoColumnSearchResults.ts:13:38) {
      info: {
        stack: "TypeError: Cannot read properties of undefined (reading 'accessibilityData')\n" +
          '    at new WatchCardHeroVideo (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/WatchCardHeroVideo.ts:18:37)\n' +
          '    at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:430:39)\n' +
          '    at new UniversalWatchCard (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/UniversalWatchCard.ts:15:34)\n' +
          '    at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:417:29)\n' +
          '    at new SecondarySearchContainer (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/SecondarySearchContainer.ts:11:28)\n' +
          '    at Function.parseItem (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:377:24)\n' +
          '    at Function.parse (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/index.ts:430:39)\n' +
          '    at new TwoColumnSearchResults (/Users/burhan/Dev/sponsortube/YouTube.js/src/parser/classes/TwoColumnSearchResults.ts:13:38)'
      },
      date: 2022-11-09T19:57:37.940Z,
      version: '2.3.3'
    }

      479 |     }
      480 |
    > 481 |     console.warn(
          |             ^
      482 |       new InnertubeError(
      483 |         `Something went wrong at ${classname}!\n` +
      484 |         `This is a bug, please report it at ${package_json.bugs.url}`, { stack: err.stack }

      at Function._Parser_printError (src/parser/index.ts:481:13)
      at Function.parseItem (src/parser/index.ts:382:27)
      at Function.parse (src/parser/index.ts:430:39)
      at new UniversalWatchCard (src/parser/classes/UniversalWatchCard.ts:15:34)
      at Function.parseItem (src/parser/index.ts:377:24)
      at Function.parse (src/parser/index.ts:417:29)
      at new SecondarySearchContainer (src/parser/classes/SecondarySearchContainer.ts:11:28)
```

### Expected behavior

Search will complete and watch_card will have accessibility data. 

### Current behavior

Function crashes

### Version

Default

### Anything else?

Here's the data passed to the WatchCardHeroVideo constructor: 
```
{
  "navigationEndpoint": {
    "clickTrackingParams": "CBkQrrEBIhMI6cK8pOah-wIVkoDCAR3H2wE-",
    "commandMetadata": {
      "webCommandMetadata": {
        "url": "/watch?v=I1qsF0WQy8c",
        "webPageType": "WEB_PAGE_TYPE_WATCH",
        "rootVe": 3832
      }
    },
    "watchEndpoint": {
      "videoId": "I1qsF0WQy8c",
      "watchEndpointSupportedOnesieConfig": {
        "html5PlaybackOnesieConfig": {
          "commonConfig": {
            "url": "https://rr5---sn-vgqsrnld.googlevideo.com/initplayback?source=youtube&oeis=1&c=WEB&oad=3200&ovd=3200&oaad=11000&oavd=11000&ocs=700&oewis=1&oputc=1&ofpcc=1&msp=1&odeak=1&odepv=1&osfc=1&id=235aac174590cbc7&ip=87.249.134.38&initcwndbps=456250&mt=1668020455&oweuc=&pxtags=Cg4KAnR4EggyNDI4NzM3Mg&rxtags=Cg4KAnR4EggyNDI4NzM3MA%2CCg4KAnR4EggyNDI4NzM3MQ%2CCg4KAnR4EggyNDI4NzM3Mg"
          }
        }
      }
    }
  },
  "title": {
    "simpleText": "Dear Twitter!"
  },
  "subtitle": {
    "simpleText": "Marques Brownlee • 3.3M views • 1 day ago"
  },
  "lengthText": {
    "accessibility": {
      "accessibilityData": {
        "label": "14 minutes, 30 seconds"
      }
    },
    "simpleText": "14:30"
  },
  "trackingParams": "CBkQrrEBIhMI6cK8pOah-wIVkoDCAR3H2wE-",
  "thumbnailOverlays": [
    {
      "thumbnailOverlayTimeStatusRenderer": {
        "text": {
          "accessibility": {
            "accessibilityData": {
              "label": "14 minutes, 30 seconds"
            }
          },
          "simpleText": "14:30"
        },
        "style": "DEFAULT",
        "type": "THUMBNAIL_OVERLAY_TIME_STATUS_TYPE_WATCH_CARD_HERO"
      }
    },
    {
      "thumbnailOverlayNowPlayingRenderer": {
        "text": {
          "runs": [
            {
              "text": "Now playing"
            }
          ]
        }
      }
    }
  ],
  "heroImage": {
    "singleHeroImageRenderer": {
      "thumbnail": {
        "thumbnails": [
          {
            "url": "https://i.ytimg.com/vi/I1qsF0WQy8c/hq720.jpg?sqp=-oaymwEcCOgCEMoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLABvV2ldwFZjW1PkuxpsSNqfpcFSA",
            "width": 360,
            "height": 202
          },
          {
            "url": "https://i.ytimg.com/vi/I1qsF0WQy8c/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLDGjB-MLTm4DkFST4P4ke8mvUWilQ",
            "width": 720,
            "height": 404
          }
        ]
      },
      "style": "SINGLE_HERO_IMAGE_STYLE_RECTANGULAR"
    }
  }
}
```  

Line 18 for WatchCardHeroVideo should be ```this.label = data.lengthText.accessibility.accessibilityData.label;```

### Checklist

- [X] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.
